### ISSUE TITLE
Skip `✔ No linting errors!`

### ISSUE THREAD

**Issue by fregante** [bluwy/renoma#2]
I think it adds a lot of noise, making actual issues harder to find.

Maybe optionally with `renoma --skip-valid`

<img width="577" alt="Screenshot" src="https://github.com/user-attachments/assets/87599df5-4141-4579-8a44-3b8a03f2ed16">

**Comment by bluwy** [comment]
Sounds good to me. Since it's doing checks and only controls the visual display, maybe `--hide-passing` would be more accurate.

Also do you mean removing the `my-lib > foo > bar:` text too? I think that's also nice to remove if you want to hide its result anyways.

**Comment by fregante** [comment]
> removing the `my-lib > foo > bar:` text too?

Yes. Just like eslint, if it passes, you output nothing. `--hide-passing` sounds good

```
$ renoma --hide-passing

mini-css-extract-plugin › schema-utils › ajv › fast-uri:
* Has lint errors same as copy-webpack-plugin › schema-utils › ajv › fast-uri

$
```

**Comment by bluwy** [comment]
I've published v0.2 which hides all passing and `Has lint errors same as *` messages altogether by default, so you'd only get lint warnings and errors only. The previous behaviour can be opt-in with `--verbose`.

**Comment by fregante** [comment]
That’s a great default!
